### PR TITLE
refine target making & modify requirements

### PR DESCRIPTION
**Pull request by MARMOTatZJU**
* refine make_densebox_target.py
  * target making code: numpy -> pytorch
  * simplified for SOT task
    * reduce to single resolution layer
* relax restrictions in requirements.txt
  * e.g. version of pytorch, opencv, numpy, etc.
* add some tips in docs

### CODE REVIEW

File: videoanalyst/data/target/target_impl/utils/debug_utils/compare_densebox_target.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    score_offset=(303 - 1 - (17 - 1) * 8) // 2,
+)
+
+target_old = make_densebox_target_old(gt_boxes, config_dict)
+target_new = make_densebox_target_new(gt_boxes, config_dict)
+
+for v_old, v_new in zip(target_old, target_new):
+    v_new = v_new.numpy()
+    err = np.abs(v_old - v_new).max()
+    print("max_err", err)
```
Comment (lzx1413) [inline videoanalyst/data/target/target_impl/utils/debug_utils/compare_densebox_target.py:21]: 可以使用np.testing.assert_allclose来做判断，更加正规一些

File: videoanalyst/data/target/target_impl/utils/debug_utils/compare_tensor.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+tensor_prefix = "center"
+
+tensor_old = np.load("{}_old.npz".format(tensor_prefix), allow_pickle=True)
+tensor_new = np.load("{}_new.npz".format(tensor_prefix), allow_pickle=True)
+
+err = np.abs(tensor_old - tensor_new).max()
+print("{} max_err: {}".format(tensor_prefix, err))
+
+from IPython import embed
+embed()
```
Comment (lzx1413) [inline videoanalyst/data/target/target_impl/utils/debug_utils/compare_tensor.py:12]: 建议使用 np.testing.assert_allclose

Comment (MARMOTatZJU) [reply videoanalyst/data/target/target_impl/utils/debug_utils/compare_tensor.py]: Done

**Comment by MARMOTatZJU** [comment]
Consider merging dev -> master (maybe main? haha BLM) after this MR.

**Pull request merged**